source/JsMaterialX/JsMaterialXCore/JsUtil.js

```diff
@@ -1,18 +1,20 @@
 // jsUtil
 addWrapper(function(Module, api) {
+    // TODO: Wrap functions and provide default args
     api.getVersionString = Module.getVersionString;
     api.createValidName = function(str, char) {
         return Module.createValidName(str, char.charCodeAt(0));
     };
 
-    api.makeVersionString = Module.makeVersionString;
     api.isValidName = Module.isValidName;
     api.incrementName = Module.incrementName;
 
     api.getVersionIntegers = function() {
         var vec = Module.getVersionIntegers();
         return vecToArray(vec);
     };
+
+    // TODO: Do we really need to map such helper functions? JS already has a String.split method.
```

Comment:
We probably should and add this as a point when we review/adjust the general implementation approach

Reply:
One thing to keep in mind is that the separator string indicates any character can be used as the separator, whereas I think the JS versoin is an exact match of the entire string.

Reply:
The JS version can take a string (in which case it looks for an exact match) or a regex. The latter can be used to achieve the same result, e.g. with `str.split(/[characters_to_search_for]/)`.

Independent of this particular function, the idea is to provide an interface that feels as native as possible to JS developers, e.g. by using the JS Array interface instead of C++ vectors. We will probably need to evaluate and discuss a couple examples like this on the way.

Reply:
Agreed, it is better to use native interfaces and as you say visit specific functions like these to see if they are useful to add. Some native utilities we're added to convert from MaterialX types to native types for the Python implementation if this is of interest.

Reply:
> @bernardkwok The PR is currently not passing the checks due to some linting issues. Since we will most likely change/remove this code soon anyway, is there a way to ignore JS linting for now? If not, I can disable the failing check manually in the affected files.

I tried ignoring this in the codacy check but I will try to disable more. It's been a bit finicky in that I have tried to restart the check manually but it may only kick in on the next push to this PR.